### DISCUSSION TITLE
ubuntu 16 + unity unresponsive to mouse (keypad) clicks

### DISCUSSION THREAD

**Discussion by bradfc** [dnschneid/crouton#4633] [Q&A]
I'm running a new Lenovo Flex 5 with Intel i3-1115G4 and just installed Ubuntu 16 + Unity.  When I get the desktop environment, the screen shows mouse movement but no response to mouse clicks.  If I close and then open the laptop lid, it looks like there is a single response, but then no more responses with the lid open.

There is also a tower of blank icons on the side panel.

At this point I cannot do anything, as I need a mouse click to select an application.

Any suggestions?

**Comment by ghost** [comment]
Does the keyboard work at all?

**Reply by bradfc** [reply]
I could type on the keyboard with no response, close the lid, then open it to discover what I had typed, but again unresponsive.

The problems went away when

- I installed Ubuntu 20.04 + xfce via crouton
- I then found a protocol on the web for downloading and installing MATE

This combination (20.04 + MATE, which is what I have on my other Linux boxes) was actually what I wanted in the first place, but  I  figured I'd be cautious with supported releases.  Perhaps Ubuntu 16 is too old to have up to date drivers.

Thanks for inquiring.
